src/my_project/utils/cleanup.py

```
@@ -3,108 +3,110 @@
+        self.cursor.execute("SELECT id, file_path FROM comics")
+        rows = self.cursor.fetchall()
+        missing = []
+        for comic_id, partial_file_path in rows:
+            full_file_path = self.config_manager.config.comicsroot.path / Path(
+                partial_file_path
+            )
+            if not os.path.exists(full_file_path):
+                missing.append((comic_id, full_file_path))
+        if len(missing) == 0:
+            logger.info("Comic database is up to date.")
+            return None
```

Ruff (0.16.1)

[warning] Do not explicitly `return None` in function if it is the only possible return value

Remove explicit `return None`

(RET501)